pubspec.yaml

```diff
@@ -5,9 +5,9 @@ version: 0.0.1
 environment:
   sdk: '>=2.18.2 <3.0.0'
 
-dev_dependencies:
-  #Static code analysis
-  flutter_lints: ^2.0.0
-  dart_code_metrics: ^5.1.0
-  lints: ^2.0.0
-  test: ^1.16.0
+dependencies:
```

Comment:
I assume that this was never used in a project as an actual package, but rather rules were copied over manually from here? Because when fetching from git development dependencies are not fetched.

I understand that these dependencies are used for static analysis and not needed for run-time, but that is solved when we use this as a dev_dependency within actual project. Given the circumstances of how it's intended to be used I think this approach is completely acceptable.

Reply:
I remember @maxpelna copying rules, but I thought that was related to experiments in his project. Maksims, can you reply to this? 😅

Reply:
Yep, this is true that in Balcia we copied rules from there so I don't need to update this repo each time something changed :D At least in the beginning there were changes each week so we decided to update rules in Balcia repo. 

Regarding dev_dependencies - in Balcia we added 
```
 flutter_lints: ^2.0.1
  lints: ^2.0.0
  dart_code_metrics: ^5.6.0
```
in dev_dependencies (probably that's why it works 😅). So this commit will help to get rid of them 👍